pkg/app/crd_app.go

```diff
@@ -114,6 +117,7 @@ func (a *CRDApp) updateApp(updateFunc func(*kcv1alpha1.App)) error {
 }
 
 func (a *CRDApp) Reconcile(force bool) (reconcile.Result, error) {
+	a.appMetrics.InitMetrics(a.app.Name(), a.app.Namespace())
```

Comment:
i think it might be better to move this call into app_reconcile.go. it'll be closer to other calls to appMetrics.

Reply:
I put it in to app_reconcile.go, however wont this be called every time a reconcile operation is performed ?
Simple test shows fine but I think that is because it adds 0 for counters.

Reply:
> Simple test shows fine but I think that is because it adds 0 for counters.

yeah it would. and adding 0 i guess is fine. factory's New method is also called every time, so same behaviour